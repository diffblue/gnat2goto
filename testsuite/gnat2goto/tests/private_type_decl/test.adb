with Private_Type_Decl; use Private_Type_Decl;

procedure Test is
begin
   Foo;
end Test;
