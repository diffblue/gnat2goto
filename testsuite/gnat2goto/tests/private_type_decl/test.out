[1] file private_type_decl.adb line 8 assertion: SUCCESS
VERIFICATION SUCCESSFUL
